strawberry/aiohttp/views.py

```diff
@@ -0,0 +1,104 @@
+import json
+from io import BytesIO
+from pathlib import Path
+
+from aiohttp import web
+from strawberry.file_uploads.data import replace_placeholders_with_files
+from strawberry.http import GraphQLHTTPResponse, process_result
+from strawberry.schema import BaseSchema
+from strawberry.types import ExecutionContext, ExecutionResult
+
+
+class GraphQLView:
+    def __init__(self, schema: BaseSchema, graphiql: bool = True):
+        self.schema = schema
+        self.graphiql = graphiql
+
+    async def __call__(self, request: web.Request) -> web.StreamResponse:
+        if request.method == "GET":
+            return await self.get(request)
+        if request.method == "POST":
+            return await self.post(request)
+        raise web.HTTPMethodNotAllowed(request.method, ["GET", "POST"])
+
+    async def get_root_value(self, request: web.Request) -> object:
+        return None
+
+    async def get_context(self, request: web.Request) -> object:
```

Comment:
do you think we should also add a `response` object here? We do that in django and asgi to allow setting cookies, not sure how that could be done in aiohttp :)

Reply:
I'll check whether this would be feasible :)

Reply:
After some research it appears uncommon to pre-create an empty response object and later add data and content-type headers. It could be done though. Is setting cookies from `get_context` a common use case? To me it feels like setting cookies would be something that is done after schema execution. Similar to the `process_result` method a `process_response` method could be introduced.

Reply:
so, what I've done and seen others do is to set cookies from a resolver, via `info.context.response.set_cookies`, not sure if there's a better way, we got the inspiration from fastapi :)

Reply:
Ah, I didn't think about setting cookies from resolvers. Good point. I'll try to implement it.

Reply:
`get_context` now takes an extra response argument. The solution works suprisingly well.

I also added some extra tests and a `RELEASE.md` file.